app/models.py

```diff
@@ -0,0 +1,14 @@
+from datetime import datetime, timezone
+from typing import Optional
+import sqlalchemy as sa
+import sqlalchemy.orm as so
+from app import db
+
+class ToDo(db.Model):
+    id: so.Mapped[int] = so.mapped_column(primary_key=True)
+    task: so.Mapped[str] = so.mapped_column(sa.String(32), index=True,
+                                                unique=True)
+    description: so.Mapped[str] = so.mapped_column(sa.String(256), index=True,
```

Comment:
remove unique on line 9, 11. todo record is fetched by id so it is not needed for task

Reply:
set to false